doc: Updated README.md & CONTRIBUTING.md file

bigdevsircole opened the pull request:
Updated README.md file with ReblendJS introduction, features & comparisons with React, async component & rendering examples, installation and usage guide, contribution guide and license info. Updated the CONTRIBUTING.md file with contribution guidelines for setup, development, and testing.

scansio reviewed (changes requested):
The changes you made are formatting, u should documentation one or two functionality/feature

bigdevsircole committed:
doc: updated README.md file & CONTRIBUTING.md file

scansio commented:
add PR description and change you title

bigdevsircole commented:
**doc: Updated README.md & CONTRIBUTING.md file**

Updated README.md file with ReblendJS introduction, features & comparisons with React, async component & rendering examples, installation and usage guide, contribution guide and license info. Updated the CONTRIBUTING.md file with contribution guidelines for setup, development, and testing.

scansio commented:
> Updated README.md file with ReblendJS introduction, features & comparisons with React, async component & rendering examples, installation and usage guide, contribution guide and license info. Updated the CONTRIBUTING.md file with contribution guidelines for setup, development, and testing.

update description which usually the first comment

scansio commented:
Thanks for your contribution
